firmware/application/ui/ui_rssi.cpp

```diff
@@ -500,4 +521,9 @@ bool RSSI::on_touch(const TouchEvent event) {
             return false;
     }
 }
+
+void RSSI::set_db(int16_t db) {
+    db_ = db;
+    set_dirty();
```

Comment:
Is the set_dirty needed here ?

Reply:
Here I'm really just asking, because in my experience bad placed or misused set_dirty are making the ui flashing